examples/complete/versions.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_version = "> 0.13"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.36, !=4.0.0, !=4.1.0, !=4.2.0, !=4.3.0, !=4.4.0, !=4.5.0, !=4.6.0, !=4.7.0, !=4.8.0"
```

Comment:
What is the intended version range to support, should this be `>= 3.36, <4.0` or `>4.8, < 5.0`

Reply:
Meant to satisfy constraint here: https://tamr.slack.com/archives/C015CHS3REV/p1701968622861829?thread_ts=1701966902.085689&cid=C015CHS3REV